app/src/main/java/org/greenstand/android/TreeTracker/models/user/User.kt

```diff
@@ -5,6 +5,8 @@ data class User(
     val wallet: String,
     val firstName: String,
     val lastName: String?,
+    val email: String?,
+    val phone: String?,
```

Comment:
The email/phone is saved as a users "wallet" ID, so whichever the user decides to use should just be saved under "wallet"

Reply:
There is currently no wallet ID in the planterInfo Entity or should I modify the columns of the planter info table to suit this.

Reply:
The field `identifier` in the PlanterInfoEntity is what we'll use as the wallet ID.